### PR TITLE
add enet package

### PR DESCRIPTION
**Pull request by wheybags**
Can be moved to a hunter-packages fork too

### CODE REVIEW

File: cmake/projects/enet/hunter.cmake [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+include(hunter_cacheable)
+
+# List of versions here...
+hunter_add_version(
+    PACKAGE_NAME
+    enet
+    VERSION
+    "1.3.13-p1"
+    URL
+    "https://github.com/wheybags/enet/archive/v1.3.13-p1.tar.gz"
```
Comment (ruslo) [inline cmake/projects/enet/hunter.cmake]: You can use https://github.com/hunter-packages/enet now.

File: examples/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+*/build/
```
Comment (ruslo) [inline examples/.gitignore]: `_builds` used:
* https://github.com/ruslo/hunter/blob/00d70c68c79899306c37e169215db2ea6f3303d7/.gitignore#L8-L13

**Comment by wheybags** [comment]
done

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.1965
* https://travis-ci.org/ingenue/hunter/builds/254739450?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
`analyze` failed

**Comment by wheybags** [comment]
PR made: https://github.com/ingenue/hunter/pull/90

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.1989
* https://travis-ci.org/ingenue/hunter/builds/256074666?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.1992
* https://travis-ci.org/ingenue/hunter/builds/256098629?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.19.37

**Pull request merged**